dd-java-agent/instrumentation/freemarker-2.3.9/src/main/java/datadog/trace/instrumentation/freemarker9/DollarVariableDatadogAdvice.java

```diff
@@ -0,0 +1,32 @@
+package datadog.trace.instrumentation.freemarker9;
+
+import datadog.trace.api.iast.InstrumentationBridge;
+import datadog.trace.api.iast.Sink;
+import datadog.trace.api.iast.VulnerabilityTypes;
+import datadog.trace.api.iast.sink.XssModule;
+import freemarker.core.DollarVariable9Helper;
+import freemarker.core.Environment;
+import net.bytebuddy.asm.Advice;
+
+public final class DollarVariableDatadogAdvice {
```

Comment:
```suggestion
public final class DollarVariableAdvice {
```

Having `Datadog` included in the name made sense if we were going to inject in the target package, but now that it is under `datadog.*`, this is no longer necessary.

Reply:
I removed the Advice class and put the DollarVariableAdvice inside the Instrumentation